scripts/update.ts

```diff
@@ -0,0 +1,748 @@
+//
+// mdn-bcd-collector: scripts/update-bcd.ts
```

Comment:
Can you use the same header as elsewhere in BCD?

I approve using the code under the BCD license, which isn't the same as in mdn-bcd-collector. @queengooborg can you also approve this?

Reply:
I replaced the headers with the common one in BCD.